### DISCUSSION TITLE
Fundar Trausta og Matthíasar - 2023/4/3

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#35] [vefsida-ummæli]
# Fundar Trausta og Matthíasar - 2023/4/3

Kvikmyndin okkar verður bráðlega tilbúin

https://www.biskupsreglan.com/frettir/funda-sig-upp

**Comment by TJ478** [comment]
Ég mun aldrei gleyma öllum fundum okkar og líka hundunum þínum Matthías
